standalone/standalone.py

```diff
@@ -540,6 +541,15 @@ def cli():
     """
 
 
+def validate_pipeline_type(value):
```

Comment:
may seem awkward placement, but it needs to appear before the @click.option for --sdg-pipeline-type, 
let me know if this should go somewhere else.